src/styles/calendar-block.scss

```diff
@@ -0,0 +1,60 @@
+$border-color: #000;
+$day-color:#eee;
+$day-active-color: #ecb9da;
+$font-size: 30px;
+$day-cell: 100px;
+$cell-gap: 1px;
+$default-padding: 10px;
+$calendar-width: 706px;
+$date-position: ($day-cell - $font-size) / 2;
```

Comment:
Let's create a separate file for variables

Reply:
good idea!